examples/usercimaimodule/imagimob/README.md

```diff
@@ -0,0 +1,60 @@
+# MicroPython ImagiMob Integration for Infineon's PSoC6 port
+
+This directory contains the imagimob generated model files and the interface for it's enablement in micropython. To access your edge ai models developed for PSoC6 based kit from a micropython application, please follow the description below.
+
+## Folder structure
+Currently the existing sample model files are generated for [Human Activity Recognition](https://developer.imagimob.com/getting-started/modus-toolbox-solution-and-Imagimob-Studio#human-activity-recognition) code from imagimob.
+
+**model.c** : Imagimob generated source code in C for your developed model in the studio.
+
+**model.h** : Imagimob generated source code in C for your developed model in the studio.
+
+**imai_mp_iface.c** : MicroPython interface for ImagiMob models.
+
+**micropython.mk** : Makefile configurations for firmware generation.
+
+## Installation
+
+Please check for the pre-requisites mentioned [here](../../ports/psoc6/README.md#Pre-requisites).
+
+## Usage
+
+Follow the steps below to generate micropython bindings for your imagimob generated model.
+
+1. The imagimob micropytthon integration is available in imai-mp-integration, so checkout to that branch after cloning this repository.
+
+        git clone https://github.com/Infineon/micropython.git
+
+        git checkout --track origin/ports-psoc6-main
+
+2. Then initialize the ModusToolbox™ environment: 
```

Comment:
You need to be already in ports/psoc6 to run this make target :)

Reply:
Fixed!